.github/workflows/aggregate_data.yml

```diff
@@ -0,0 +1,30 @@
+name: Run aggregate_data.py
+
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  build:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout repo content
+        uses: actions/checkout@v3 # checkout the repository content to github runner.
```

Comment:
use v3 or v4 to skip the warning that will become an error this summer
![image](https://user-images.githubusercontent.com/33978275/210649439-61d1e21e-7976-4bfc-b7ed-68528ef0a2de.png)